setting-up-tugboat/index.md

```diff
@@ -0,0 +1,510 @@
+  setting:\*\*** any secrets in your Preview will be accessible by the owner of
+  the forked repository.
+
+You can also specify the account from which comments are posted to BitBucket in
+this section. For info on customizing this, see:
+[Add a Tugboat Bot to your team](../administering-tugboat-crew/index.md#add-a-tugboat-bot-to-your-team).
+
+### Adding a link to a git provider
+
+Need to add a git provider to your Tugboat account? No problem! Whether you
+created your initial Tugboat account with an email and now want to add a git
+provider, or whether you're adding your second or third git provider - you can
+easily connect your Tugboat account with additional git providers.
+
+1. Click the User drop-down in the upper right-hand corner of the Tugboat
+   dashboard, and select **Profile**;
+2. Click the **+ Connect Account** link;
+3. Select the git provider whose account you'd like to connect;
+4. Follow the instructions to connect to
+   [GitHub](#how-do-i-link-my-tugboat-to-github),
+   [GitLab](#how-do-i-link-my-tugboat-to-gitlab) or
+   [BitBucket](#how-do-i-link-my-tugboat-to-bitbucket).
+
+Click the User drop-down in the upper right-hand corner of the Tugboat
+dashboard, and select **Profile**.
+
+![Go to User drop-down, and select Profile](_images/go-to-user-profile.png)
+
+Click the **+ Connect Account** link.
+
+![Click the + Connect Account link](_images/click-connect-account-link.png)
+
+Select the git provider whose account you'd like to connect.
+
+![Select a git provider](_images/select-a-git-provider.png)
+
+## Create a new project
+
+Once you've connected Tugboat to your preferred git provider, it's time to
+create a new project!
+
+- [How to create a project](#how-to-create-a-project)
+- [Things to know about Tugboat projects](#things-to-know-about-tugboat-projects)
+- [Add repos to the project](#add-repos-to-the-project)
+
+Once you hit the **Create Project** button, it's anchors away!
+
+### How to create a project
+
+The first time you sign into Tugboat, you'll go directly to the Create New
+Project screen.
+
+After you've already got projects, you can add new ones by going to the Tugboat
+Dashboard and selecting **Create New Project**.
+
+### Things to know about Tugboat projects
+
+When you're creating a project, there are a few things to keep in mind:
+
+- A Tugboat project is a collection of any number of repos across git providers
+- Tugboat's pricing tiers and billing are specified on the project level, not
+  the number of repositories
+- Users are managed on a per project basis
+
+With this in mind, you may find yourself sailing on stormy seas if you're
+building a lot of Previews (or resource-intensive Previews) on a low-level
+project tier. If that's the case, consider structuring your projects by putting
+repos requiring resource-intensive Preview builds in their own projects, while
+other repos' Preview builds proceed in a different project without performance
+hits.
```

Comment:
> With this in mind, you may find yourself sailing on stormy seas if you're
 building a lot of Previews (or resource-intensive Previews) on a low-level
 project tier. If that's the case, consider structuring your projects by putting
 repos requiring resource-intensive Preview builds in their own projects, while
 other repos' Preview builds proceed in a different project without performance
 hits.

A preview’s performance is not directly affected by the other previews in a project.
The exception being whether there is enough disk space.